x-pack/plugins/monitoring/server/plugin.ts

```diff
@@ -64,7 +63,7 @@ const wrapError = (error: any): CustomHttpResponseOptions<ResponseError> => {
   };
 };
 
-export class Plugin {
+export class MonitoringPlugin implements Plugin<any, void, PluginsSetup, PluginsStart> {
```

Comment:
why `any`? API interface is defined. @chrisronline Where `getKibanaStats` is used?